jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/BuildDockerTask.java

```diff
@@ -164,6 +165,13 @@ public void buildDocker()
       throw new GradleException(
           "container.volumes is not an absolute Unix-style path: " + ex.getInvalidVolume(), ex);
 
+    } catch (InvalidFilesModificationTimeException ex) {
+      throw new GradleException(
+          "container.filesModificationTime should be an ISO 8601 date-time (see "
```

Comment:
`jib.container.filesModificationTime`

Reply:
That's better, but all the other error messages around this code do not start with `jib.container`, so I think I should go without it for consistency here.